src/platform/plugins/shared/unified_doc_viewer/public/components/observability/traces/doc_viewer_span_overview/span_overview.tsx

```diff
@@ -54,7 +54,10 @@ export function SpanOverview({
       >
         <EuiPanel color="transparent" hasShadow={false} paddingSize="none">
           <EuiSpacer size="m" />
-          <SpanSummaryTitle name={parsedDoc[SPAN_NAME_FIELD]} id={parsedDoc[SPAN_ID_FIELD]} />
+          <SpanSummaryTitle
+            name={{ value: parsedDoc[SPAN_NAME_FIELD], field: SPAN_NAME_FIELD }}
+            id={{ value: parsedDoc[SPAN_ID_FIELD], field: SPAN_ID_FIELD }}
```

Comment:
Q: Why do the Span and Transaction title components need to know the field name? Could both components just receive the id and name values instead?

```suggestion
            name={parsedDoc[SPAN_NAME_FIELD]}
            id={parsedDoc[SPAN_ID_FIELD], field: SPAN_ID_FIELD]}
```

Reply:
I figure we might not want to hardcode the field names if we are going to make things work with OTEL in the future. That said, that mechanism is not known yet, so can always revisit it. But the field names are necessary for the `FieldActionsPopover` to work.